playbooks/roles/mariadb/tasks/main.yml

```diff
@@ -0,0 +1,93 @@
+---
+#
+# edX Configuration
+#
+# github:     https://github.com/edx/configuration
+# wiki:       https://github.com/edx/configuration/wiki
+# code style: https://github.com/edx/configuration/wiki/Ansible-Coding-Conventions
+# license:    https://github.com/edx/configuration/blob/master/LICENSE.TXT
+#
+#
+#
+# Tasks for role mariadb
+# 
+# Overview:
+# 
+#
+# Dependencies:
+#
+# 
+# Example play:
+#
+#
```

Comment:
all these nits are from autogenerated stuff from the `create_role` role.  Also, since this is just pulling from open source we should probably work on them there, maybe make an issue on `edx/configuration`?  I don't really know how to best deal with that.